### PR TITLE
Updated blog post design.

### PR DESCRIPTION
**Pull request by zacharytamas**

### CODE REVIEW

File: _posts/2016-11-15-announcing-polygigs.md [hunk 1]
```diff
@@ -19,6 +21,6 @@ Because of this we see an increasing demand for developers specifically with Pol
 
 To help with this transition, today we are pleased to announce the launch of [Polygigs](https://polygigs.com/), a simple Polymer job board. Though the website is very basic for now we have a long list of features and functionality we’re planning to implement.
 
-The website is of course built with Polymer as well and we’re keeping its source code open and available on our GitHub at: https://github.com/NorthernLogic/polygigs
+The website is of course built with Polymer as well and we’re keeping its source code open and available on our GitHub at: [https://github.com/NorthernLogic/polygigs](https://github.com/NorthernLogic/polygigs)
```
Comment (adamduren) [inline _posts/2016-11-15-announcing-polygigs.md]: Maybe the link should be on the word Github?

File: assets/styles/blog.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (adamduren) [inline assets/styles/blog.scss]: Where does this get imported?

File: _posts/2016-11-15-announcing-polygigs.md (hunk 1 above)
Comment (zacharytamas) [inline _posts/2016-11-15-announcing-polygigs.md]: Perhaps. It does flow strangely.

Comment (adamduren) [inline _posts/2016-11-15-announcing-polygigs.md]: Maybe:

The website is of course built with Polymerand we’re keeping it open source. It's availablee on our [GitHub](https://github.com/NorthernLogic/polygigs) repo.

**Comment by adamduren** [comment]
Is this good to merge?

**Comment by zacharytamas** [comment]
I can't remember, sadly. About to check into this.

**Comment by zacharytamas** [comment]
No this is not ready at all and would be dangerous. Working on it now.

**Pull request merged**